inspirehep/modules/records/api.py

```diff
@@ -72,9 +72,19 @@ def create(cls, *args, **kwargs):
                 records.
         """
         files_src_record = kwargs.pop('files_src_record', None)
+        skip_files = kwargs.pop(
+            'skip_files',
+            current_app.config.get('RECORDS_SKIP_FILES', False),
+        )
```

Comment:
Uhm, the fact that the value of the parameter takes precedence means that the configuration variable is ignored unless the method was explicitly using the parameter, which doesn't really work as a workaround... probably it should be the opposite.

Reply:
The global config is ignored only when the parameter is passed, not the other way around (that allows locally oberriding the config value, if need be).

Reply:
Whoops, right, I was wrong. My problem was that I put the configuration file in the wrong place, in fact...